generators/server/templates/quarkus/src/main/resources/application.properties.ejs

```diff
@@ -96,7 +96,18 @@ quarkus.oidc.authentication.cookie-path=/
 quarkus.oidc.authentication.redirect-path=/login/oauth2/code/oidc
 quarkus.oidc.authentication.restore-path-after-redirect=false
 
+
+jhipster.oidc.logout-url=http://localhost:9080/auth/realms/jhipster/protocol/openid-connect/logout
```

Comment:
I think you don't need this. I removed the explicit logout path dependency in my last check-in